x-pack/test/functional/es_archives/security_solution/timestamp_override_3/mappings.json

```diff
@@ -3,6 +3,7 @@
   "value": {
     "index": "myfakeindex-3",
     "mappings": {
+      "dynamic": "strict",
```

Comment:
NOTE: Just added this to make sure I didn't add any bugs so it will blow up if it cannot insert the correct data.